client/my-sites/customer-home/cards/features/quick-start/index.js

```diff
@@ -0,0 +1,113 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+import { Button, Card } from '@automattic/components';
+import { connect } from 'react-redux';
+import 'moment-timezone';
+
+/**
+ * Internal dependencies
+ */
+import HappinessEngineersTray from 'components/happiness-engineers-tray';
+import CardHeading from 'components/card-heading';
+import {
+	withAnalytics,
+	composeAnalytics,
+	recordTracksEvent,
+	bumpStat,
+} from 'state/analytics/actions';
+import { navigate } from 'state/ui/actions';
+import { getSelectedSiteId, getSelectedSiteSlug } from 'state/ui/selectors';
+import QueryConciergeInitial from 'components/data/query-concierge-initial';
+import getConciergeNextAppointment from 'state/selectors/get-concierge-next-appointment';
+import { withLocalizedMoment } from 'components/localized-moment';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+const QuickStart = ( {
+	moment,
+	nextSession,
+	reschedule,
+	siteId,
+	siteSlug,
+	translate,
+	viewDetails,
+} ) => {
+	if ( ! siteId ) {
+		return null;
+	}
+	{
+		nextSession ? (
+			<Card className="quick-start next-session">
+				<HappinessEngineersTray />
+				<CardHeading>{ translate( 'Your scheduled Quick Start support session' ) }</CardHeading>
+				<table>
+					<thead>
+						<tr>
+							<th>{ translate( 'Date' ) }</th>
+							<th>{ translate( 'Time' ) }</th>
+						</tr>
+					</thead>
+					<tbody>
+						<tr>
+							<td>{ moment( nextSession.beginTimestamp ).format( 'LL' ) }</td>
+							<td>
+								{ moment.tz( nextSession.beginTimestamp, moment.tz.guess() ).format( 'LT z' ) }
+							</td>
+						</tr>
+					</tbody>
+				</table>
+				<Button onClick={ () => viewDetails( siteSlug ) }>{ translate( 'View details' ) }</Button>
+				<Button
+					className={ 'quick-start__reschedule' }
+					href={ `/me/concierge/${ siteSlug }/book` }
+					onClick={ () => reschedule( siteSlug, nextSession.id ) }
+					borderless
+				>
+					{ translate( 'Reschedule' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **21** times:
`translate( 'Reschedule session' )` *ES Score: 8*